picoquictest/tls_api_test.c

```diff
@@ -779,6 +779,8 @@ static int tls_api_one_sim_round(picoquic_test_tls_api_ctx_t* test_ctx,
                     /* useless test, but makes it easier to add a breakpoint under debugger */
                     ret = -1;
                 }
+
+                free(packet);
```

Comment:
Isn't the data already freed at line 743?

Reply:
No, the packet is created in line 762